openhands/core/config/utils.py

```diff
@@ -385,7 +385,7 @@ def get_parser() -> argparse.ArgumentParser:
     parser.add_argument(
         '-n',
         '--name',
-        default='default',
+        default='',
```

Comment:
Was this necessary for some reason? It's the same default value as [this one](https://github.com/All-Hands-AI/OpenHands/blob/2ec2f2538fa403a6df8bdda48c2d35afb4bb9c6f/openhands/controller/agent_controller.py#L85), which seems to make some sense

Reply:
We should probably remove that one too

This causes headless mode to reuse the previous session's data, unless the user specifies `-n`. So this is pretty broken as-is

Reply:
OK 🤔